examples/vision/adamatch.py

```diff
@@ -128,7 +128,7 @@
 
 # Initialize `RandAugment` object with 2 layers of
 # augmentation transforms and strength of 5.
-augmenter = RandAugment(num_layers=2, magnitude=5)
+augmenter = RandAugment(value_range=(0, 255), augmentations_per_image=3, magnitude=0.5)
```

Comment:
augmentations_per_image=2 to reproduce your original, up to you how you want to implement this though.

Reply:
Ah, yes I see, the comment above it should be adapted then as well. I give it a try to run it with `augmentations_per_image=2` and see if I am more lucky in getting good results quickly this time, if not I guess I would leave it at `3`

Reply:
If you have issues please let me know!